deploy_pulp3/pulp3_systemd/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for pulp3
+pulp_user: pulp
+pulp_venv : "/home/pulp/pulpvenv"
```

Comment:
How about setting this at runtime with the `getent` and `set_fact` modules? The Pulp user's home directory could be in any number of places.